package.json

```diff
@@ -23,54 +23,62 @@
     "url": "git+https://github.com/Microsoft/PowerBI-visuals-sunburst.git"
   },
   "devDependencies": {
-    "@types/d3-hierarchy": "^3.1.2",
-    "@types/d3-selection": "^3.0.5",
-    "@types/d3-shape": "^3.1.1",
-    "@types/jasmine": "^4.3.5",
-    "@types/jasmine-jquery": "^1.5.34",
-    "@types/jquery": "3.5.16",
-    "@types/karma": "6.3.4",
-    "@types/lodash": "4.14.196",
-    "@types/webpack": "5.28.1",
+    "@types/d3-hierarchy": "^2.0.0",
+    "@types/d3-selection": "^2.0.0",
+    "@types/d3-shape": "^2.0.0",
+    "@types/jasmine": "^3.6.2",
+    "@types/jasmine-jquery": "^1.5.33",
+    "@types/jquery": "3.5.5",
+    "@types/karma": "^6.3.4",
+    "@types/lodash": "4.14.166",
+    "@types/webpack": "4.41.25",
```

Comment:
Why did you revert these changes?

Reply:
I ran the tests and they failed, one of the errors were "function getJSONFixtures() not found on jasmine" so it seemed the configuration was wrong. I tried figuring out what went wrong, but in the end it was easier to just revert the changes

Reply:
@SanderGi getJSONFixtures fail is related to jasmine-query to load capabilities.json file. You can copy the code from [here](https://github.com/microsoft/powerbi-visuals-sankey/blob/main/test/visualTest.ts#L785)